Lesson2/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson2
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
For what we have this project ? :)

Reply:
I don't know. I don't see this project in VS

Reply:
Lets have a call, i'll try to help